upi/openstack/inventory.yaml

```diff
@@ -54,13 +54,13 @@ all:
       # Be aware that the 10 and 11 of the machineNetwork will
       # be taken by neutron dhcp by default, and wont be available.
       # This value will be overwritten by the network.yaml playbook.
-      os_apiVIP: "{{ os_subnet_range | ansible.utils.next_nth_usable(5) }}"
+      os_apiVIP: "{{ os_subnet_range | next_nth_usable(5) }}"
```

Comment:
why do you remove all modules namespaces here?

Reply:
Because ansible utils we get from the openstack-17-for-rhel-9-rpms is too old (version 2.3.0-2.el9ost) and doesn't have any of the needed filters added in v2.5.